### DISCUSSION TITLE
Hob remote Poweroff impossible?

### DISCUSSION THREAD

**Discussion by giggls** [hcpy2-0/hcpy#113] [Q&A]
Hello,

looking for non cloud options for controlling my Siemens devices I ended up here. 

The Software seems to work so far. But I have a question:

Is it really impossible to remote Power-off my Hob?

The model I am talking about is EX607NYV6E

I mean this is likely one of the most wanted remote features. A lot of people will definitely want be able to resolve the infamous mulling about "did we power off the hob" while travelling to holiday destinations.

I am able to set the hobs timer like this:
mosquitto_pub -L 'mqtt://localhost/homeconnect/hob/set' -m '{"uid":565,"value":60}'

so communication works, but looking at config/devices.json "BSH.Common.Setting.PowerState" access is just "read" not "readWrite".

Any possibility to change this?

Attached is the devices.json regarding my hob.

[devices.json](https://github.com/user-attachments/files/18131588/devices.json)

**Comment by Hypfer** [comment]
Can you turn it off via the official app?
If no, then it's very likely that it's not possible.

**Reply by giggls** [reply]
I already expected something like this as I did not succeed with the official app either.

**Comment by Meatballs1** [comment]
Can you turn individual zones on and off?

**Comment by Asmir1975** [comment]
Hi, on/off is local-only. Stopwatch time, child lock and alarm clock work via the app, while other settings still need confirmation on the hob. This is Siemens/BSH design.
Model: EX851LYV5E/60

<img width="963" height="1024" alt="Snap IOS App" src="https://github.com/user-attachments/assets/1b67ad83-019f-427f-81a7-708bd5e74953" />
